lib/travis/build/addons/apt_retries.rb

```diff
@@ -0,0 +1,34 @@
+require 'travis/build/addons/base'
+
+module Travis
+  module Build
+    class Addons
+      class AptRetries < Base
+        SUPER_USER_SAFE = true
+
+        def before_configure?
+          config
+        end
+
+        def before_configure
+          sh.echo "Configuring default apt-get retries", ansi: :yellow
+          set_retries = <<-EOF
```

Comment:
We can safely use `<<~HEREDOC`, iirc, so that this whole string may be safely indented.  The APT configs don't care about blankspace either, fwiw.

Reply:
I was not aware of `<<~`.